mantidproject/mantid
# Use direct array access to manipulate vtkUnstructuredGrid

Description of work.

This updates the vtkMDHexFactory to use direct array access. We founder earlier (see PR #14236) that this is several times faster than using the member function `InsertNextPoint`.

**To test:**

Verify one can still load and view a MDEventWorkspace. 



This PR has no associated issue number.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
